Created Capstone Project Using HTML CSS & JavaScript

Mike111222 opened the pull request:
I have done the following changes and I want to implement these:

- Added Linters setup
- Created index.html file for Home page
- Created about,html file for About page
- Created styles.css
- Created script.js for hamburger menu code and loading the speaker section code on run time via JS.
- Updated README.md
- Added a Loom Video link to README.md for Live Demo of my project
- Added images and logos
- Followed the kind reviewer's suggestions and updated accordingly.
- Added JS Code for Show More Button and Loaded only 2 speakers in Mobile Version.

Mike111222 committed:
Add Linters Setup

Mike111222 committed:
Add images and other assets

Mike111222 committed:
Create about.html

Mike111222 committed:
Create index.html

Mike111222 committed:
Create script.js

Mike111222 committed:
Create styles.css

Mike111222 committed:
Update README.md

Mike111222 committed:
Update README.md

Mike111222 committed:
Update about.html

Mike111222 committed:
Update README.md

tufoinnkuo10 commented on README.md:
```diff
@@ -1,2 +1,227 @@
+# 📖 [Capstone Project] <a name="about-project"></a>
+
+
+**[Capstone Project]** is the first milestone to get new Micronauts onboard with Figma, Own Responsive Website, Git, GitHub and Linters and how to fulfill the customer requirements.
```
- [optional] you can add your site title instead of 'capstone project', also i can see you have two project title, one at the top and this one. Please, kindly use one for a professional README file and please ensure the description is about what you built.

Mike111222 replied:
Dear Sir, thanks a lot for pointing it out for me, I have fixed it.

tufoinnkuo10 commented on README.md:
```diff
@@ -1,2 +1,227 @@
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<a href="https://www.loom.com/share/e5685f1ec8404da488b524c1d9e6e79a" target="_blank">Mike's Capstone Project Live Demo Link</a>
```
- [ ] Great job with the loom video. Unfortunately, you will be given an opportunity to present it later. kindly add a deployed link here instead. Ensure it is deployed properly using GitHub Pages and all pages, links, and images work as expected

Mike111222 replied:
Dear Sir, thanks a lot for pointing it out for me, I have fixed it. I have the deployed it and the link is in the README.md

Efiamotu-1 replied:
@Mike111222 as requested by the previous reviewer your live demo link is required to display the deployed version of your project once it is clicked, however it currently does not display this rather it takes one to the github repo for this project. please kindly add the deployed version link before requesting for your next review :+1: 
-  [ ] It is important to note that you are to deploy using `gh pages` as requested by the previous reviewer :+1:

tufoinnkuo10 commented on about.html:
```diff
@@ -0,0 +1,128 @@
+  <nav class="mini-nav on-desktop">
+    <ul class="mininav-menu">
+      <li class="mininav-item"><img src="/img/facebook.svg" alt="facebook"></li>
+      <li class="mininav-item"><img src="/img/twitter.svg" alt="twitter"></li>
```
- [ ] nice job. Please ensure your work is responsive from mobile to the desktop version
![Screenshot (1082)](https://github.com/Mike111222/CapstoneProject/assets/65298048/2a847667-6c67-4cce-812c-eb48314b63c1)

Mike111222 replied:
Thanks a lot for you kind feedback sir. I have implemented the changes suggested with responsive units.

tufoinnkuo10 commented on about.html:
```diff
@@ -0,0 +1,128 @@
+    </ul>
+  </nav>
+  <header id="myheader">
+    <a class=" logo on-desktop" href="#home"><img src="/img/logo.jpg" class="logo-nav" alt="logo"></a>
```
- [ ] when in your about page, the logo doesn't take you back to the home page for the desktop version. you can fix that with this `href="/index.html"` or '/'
- [ ] for the mobile menu, kindly add a link that can take you to the home page when in the about page. nice work

Mike111222 replied:
Dear Sir, I have fixed it. Thanks a lot for pointing it out for me.

Mike111222 replied:
I have done the changes as suggested. When a user clicks on Program in mobile menu, it takes the user to the home page

tufoinnkuo10 commented on about.html:
```diff
@@ -0,0 +1,128 @@
+          </div>
+        </div>
+      </div>
+    </div>
```
- [ ] you are missing the partner section of the mobile about page. please kindly follow the design
yours
![Screenshot (1080)](https://github.com/Mike111222/CapstoneProject/assets/65298048/eb6cd77f-c141-4498-babc-6783db2f3e45)
Behance design
![Screenshot (1079)](https://github.com/Mike111222/CapstoneProject/assets/65298048/e11b4678-d152-499a-ba48-6d136839a2fa)

Mike111222 replied:
I have done the changes suggested by you sir. Thanks

tufoinnkuo10 commented on index.html:
```diff
@@ -0,0 +1,158 @@
+  </nav>
+  <header id="myheader">
+    <!-- <a class="logo" href="#home">My Logo</a> -->
+    <a class=" logo on-desktop" href="#home"><img src="/img/logo.jpg" class="logo-nav" alt="logo"></a>
```
- [ ] please kindly align your logo with the text in the main section like in the original design.
- [ ] kindly add the font size of the main heading and make it bolder as well
- [ ] please add some padding top and bottom below the date and above the main heading, so the main section can follow the design.
yours
![Screenshot (1075)_LI](https://github.com/Mike111222/CapstoneProject/assets/65298048/1a011882-40f7-4b4d-a472-04e119a67278)
original
![Screenshot (1071)](https://github.com/Mike111222/CapstoneProject/assets/65298048/51792ea6-d9de-4f9d-b8d4-90d9b560db45)

Mike111222 replied:
I have implemented all of the above-mentioned changes. Thanks a lot again

tufoinnkuo10 commented on index.html:
```diff
@@ -0,0 +1,158 @@
+        <div><img src="/img/program_icon_02.png" alt=""></div>
+        <h3 class="program-subheading">
+          HG Exhibition
+        </h3>
```
- [ ] Great job. From the original design, the cards have their separate background color. please implement this.
- [ ] you are missing the underlined text below the cards. kindly add that (see the whole program)
yours
![Screenshot (1076)](https://github.com/Mike111222/CapstoneProject/assets/65298048/e738f9ca-fd15-4640-b222-ce9026af29a4)
original
![Screenshot (1011)](https://github.com/Mike111222/CapstoneProject/assets/65298048/fe838291-c23a-4095-a746-8f2cf2dc2abe)

Mike111222 replied:
I have implemented the above-mentioned changes sir. Thanks for the kind info on all these.

tufoinnkuo10 commented on index.html:
```diff
@@ -0,0 +1,158 @@
+        <h3 class="program-subheading">
+          Forum
+        </h3>
+        <p class="program-info">
```
- [ ] please add the cards their separate background like i mentioned for the desktop version.
![Screenshot (1083)](https://github.com/Mike111222/CapstoneProject/assets/65298048/0b9f83d3-ce2d-4153-9a51-eb6cc632e454)

Mike111222 replied:
I have implemented these changes sir. Thanks for the kind info and great help.

tufoinnkuo10 commented on index.html:
```diff
@@ -0,0 +1,158 @@
+    <button type="button" class="join-btn">Join HG Global Summit 2023</button>
+  </section>
+  <!-- the Speakers page -->
+  <section><h3 class="speaker-headline">Featured Speakers</h3></section>
```
- [ ] kindly add some padding to the speaker heading top and bottom. also please add the font size 
- [ ] like in the original design, add margin left and right to the speaker section.
- [ ] kindly add the red line under the partner section heading. nice work.
your design
![Screenshot (1078)_LI](https://github.com/Mike111222/CapstoneProject/assets/65298048/d0f01f87-8380-467c-968e-0e5b13b6993f)
original
![Screenshot (1078)](https://github.com/Mike111222/CapstoneProject/assets/65298048/611e25af-818c-43dd-9f92-973674aa8e7b)

Mike111222 replied:
I have done the changes you suggested sir. Thanks again

tufoinnkuo10 commented on index.html:
```diff
@@ -0,0 +1,158 @@
+  <!-- the Speakers page -->
+  <section><h3 class="speaker-headline">Featured Speakers</h3></section>
+  <div id="speakers">    
+    
```
- [ ] in the mobile version, only two speakers are visible unless you want to implement the see more functionality. if not kindly follow the design.
yours
![Screenshot (1084)](https://github.com/Mike111222/CapstoneProject/assets/65298048/5a18b34e-ac09-461e-925f-69798237ed1d)
main design
![Screenshot (1085)](https://github.com/Mike111222/CapstoneProject/assets/65298048/81e07c47-e630-4898-b33b-12a54a17567c)

Mike111222 replied:
I have implemented the suggested changes with JavaScript sir. Thanks for the kind inputs.

tufoinnkuo10 commented on index.html:
```diff
@@ -0,0 +1,158 @@
+  </section>
+  <footer>
+    <div class="cc"><img src="/img/logo.jpg" alt="logo">
+      <p class="rights">2023 HopeGrower Gold Pakistan. Some Rights Reserved.</p>
```
- [ ] please the footer section of the desktop homepage has a white background.

Mike111222 replied:
I have followed your instructions and implemented the changes you suggested sir.

tufoinnkuo10 reviewed (changes requested):
Hi @Mike111222 

There are some issues that you still need to work on  but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Design well followed

Required Changes ♻️
Check the comments under the review

![tufoin](https://media.giphy.com/media/KG5KpLlr1UEPvwgkiM/giphy.gif)

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

Mike111222 committed:
Update README.md

Mike111222 committed:
Update styles.css

Mike111222 committed:
Update about.html

Mike111222 committed:
Update index.html

Mike111222 committed:
Update script.js with Show More Button

Mike111222 committed:
Update styles.css

Mike111222 committed:
Update script.js

Mike111222 committed:
Update styles.css

Mike111222 committed:
Update index.html

Mike111222 committed:
Update styles.css

Mike111222 committed:
Update README.md

Mike111222 committed:
Update about.html

Mike111222 committed:
Update index.html

Mike111222 committed:
Update styles.css

Mike111222 committed:
Update styles.css

Efiamotu-1 reviewed (changes requested):
Hi @Mike111222 , 

![](https://camo.githubusercontent.com/a06454929d29b512682cd8b375641b59076b9e5fe1a367808e685e5bcb818de3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e32466d5957466c5a6a45794d7a49314f4455784d5455775a44686b595459355932526c4e6a4d334e7a4e6c4f574530596a4a684d695a6c634431324d563970626e526c636d35686246396e61575a7a583264705a6b6c6b4a6d4e305057632f7a3370696f6b77663057504838314d4f68752f67697068792e676966)

Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the first reviewer's review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@Efiamotu-1](https://img.shields.io/github/followers/Efiamotu-1?label=Habeeb-Efiamotu-Musa&style=social)](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[![Habeeb Efiamotu Musa](https://img.shields.io/badge/linkedin-@Habeeb-crimson)](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [![@EFYAMOTU](https://img.shields.io/twitter/follow/omarramoun?label=Habeeb-Efiamotu-Musa&style=social)](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [![Habeeb Efiamotu Musa ](https://img.shields.io/badge/slack-@Habeeb-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mike111222 committed:
Update README.md

Mike111222 commented:
Respected sir, I have updated the README.md and implemented the changes you suggested and checked the deployment. Sir, this is my LAST code review (No Extra Review Left either) and I really dont want to repeat the block please.

topeogunleye reviewed (approved):
Hi @Mike111222, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

> ## (Highlights) Good Points: 💪 
> * Added deploy link for live demo as suggested by the previous reviewer [here](https://github.com/Mike111222/HopeGrower-Gold/pull/1#discussion_r1204754666) ✔️ 
> * Live demo link goes to working version of website ✔️ 
> * Live demo deployed using GitHub pages as suggested ✔️ 

![çrpik-kurdî-impressive](https://user-images.githubusercontent.com/52934128/180656067-c6788183-0594-4658-94e3-3d2523b6db08.gif)



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification.
 
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
